### PR TITLE
added instructions to setup the project locally

### PR DESCRIPTION
**Pull request by sadath2001**
I added a detailed recorded video on how to setup the project locally and also added a few links that are related to the project.

### CODE REVIEW

**Comment by sadath2001** [comment]
Also, i would like to add GSOC extended and Hactober fest as labels to this PR,  but i presume i dont have access to it, could you please do the needful. thanks

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -8,7 +8,27 @@ Welcome to the TastyTrails project! We're excited to have you contribute and hel
...
+- [How to setup project locally](https://youtu.be/eJ0p6rYek-8) | [How to generate github API key](https://www.youtube.com/watch?v=iLrywUfs7yU)
+- Here is a list of required packages to install as mentioned in the video
+  - **Mongo and its packages**: 
+  MongoDB Community Edition – https://www.mongodb.com/try/download/community
+  Mongodb shell – https://www.mongodb.com/try/download/shell
+  Mongodb compass – https://www.mongodb.com/try/download/compass
+  Make sure to check if mongo is up and running (You can refer youtube for troubleshooting)
+  - **Npm packages**: 
+    To setup initial required packages for both front-end and backend, go to respective folders i.e */frontend and /backend* and execute the below command
+    > `npm init`
```
Comment (AlfiyaSiddique) [inline CONTRIBUTING.md:21]: `npm init` is used to initiate npm.
I think this should be ` npm i install` to install the packages. Please Edit.

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -8,7 +8,27 @@ Welcome to the TastyTrails project! We're excited to have you contribute and hel
...
+    
+    To install react router go to /frontend and execute the below command
+    > `npm i react-router-dom`
```
Comment (AlfiyaSiddique) [inline CONTRIBUTING.md:24]: Once `npm i install` is executed no need to run this command.

**Comment by AlfiyaSiddique** [comment]
@sadath2001 3 minor fix, rest looks good to me

**Review by sadath2001 — COMMENTED** [review]
Made the changes accordingly

**Comment by sadath2001** [comment]
hi @AlfiyaSiddique , i have made the changes

**Review by AlfiyaSiddique — APPROVED** [review]

**Comment by AlfiyaSiddique** [comment]
Thanks for the contribution @sadath2001

**Comment by sadath2001** [comment]
Your most welcome 🤗

**Comment by AlfiyaSiddique** [comment]
issue #22

**Pull request merged**